bitnami/external-dns/README.md

```diff
@@ -118,6 +118,9 @@ The following table lists the configurable parameters of the external-dns chart
 | `google.serviceAccountSecret`          | When using the Google provider, specify the existing secret which contains credentials.json (optional)                                                                                                          | `""`                                                    |
 | `google.serviceAccountSecretKey`       | When using the Google provider with an existing secret, specify the key name (optional)                                                                                                                         | `"credentials.json"`                                    |
 | `google.serviceAccountKey`             | When using the Google provider, specify the service account key JSON file. (required when `google.serviceAccountSecret` is not provided. In this case a new secret will be created holding this service account | `""`                                                    |
+| `hetzner.secretName`                   | When using the Hetzner provider, specify the existing secret which contains your token. Disables the usage of `hetzner.token` (optional)                                                                        | `""`                                                    |
+| `hetzner.secretKey`                    | When using the Hetzner provider with an existing secret, specify the key name (optional)                                                                                                                        | `"hetzner_token"`                                       |
+| `hetzner.token`                        | When using the Hetzner provider, specify your token here. (required when `hetzner.secretName` is not provided. In this case a new secret will be created holding the token.)                                    | `""`                                                    |
```

Comment:
Please format the table properly

Reply:
I don’t know hat you’re referring to. The table is formatted properly, the spacing everywhere matches up.

Reply:
It is just to maintain the table format. Anyway, don't worry, we usually format it when we create a new PR and it is not an issue

Reply:
Thanks for merging! I got the table format, but that’s what confused me about your request. The format is correct, it just looks weird in the small preview GitHub shows us here.

In the actual files, it looks like this:

![screenshot of the raw file](https://user-images.githubusercontent.com/7683567/100099313-8d295880-2e5f-11eb-8ad0-b17df1fad3e3.png)